content/en/docs/contribute/_index.md

```diff
@@ -1,14 +1,14 @@
 ---
 content_type: concept
-title: Contribute to Kubernetes docs
+title: Contribute to K8s docs
```

Comment:
Will this trigger another discussion about a guide on when to use "k8s" and when not to?
As official document, I'm a little concerned about this.

Reply:
We can remove this one if it's a concern.